content/blog/20241129_fluentd-v1.18.0-has-been-released.md

```diff
@@ -0,0 +1,150 @@
+# Fluentd v1.18.0 has been released
+
+Hi users!
+
+We have released v1.18.0 on 2024-11-29. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1180---20241129).
+
+This release is a new release of v1.18 series.
+In this release, we added zero-downtime-restart as a key feature, enhanced some features and fixed bugs.
+
+## Enhancement
+
+### Add zero-downtime-restart feature
+
+In this release, we have added the feature to update the Fluentd process with zero downtime for Linux and macOS environments (not supported on Windows).
+Fluentd had a feature called by graceful reload that reloads the configuration without restart.
+However, the graceful reload feature has a problem that causes a downtime, it cannot receive the data from network using `in_udp`, `in_tcp`, and `in_syslog` plugins during reloading configuration.
+This feature could only be used to reload the configuration, and could not be used to upgrade Fluentd.
+We think it had been difficult to use in a production environment by these restrictions.
+
+The zero-downtime-restart feature has been introduced to solve the problem of the graceful reload.
+This feature supports Fluentd upgrades and configuration reloads with zero downtime.
+
+This feature can be used by sending a `SIGUSR2` signal to the supervisor process,
+and this changes behavior `SIGUSR2` from graceful reload to zero-downtime-restart feature on Linux and macOS environments.
```

Comment:
How about something like this?

* From the user's perspective, it would be easier to read if there was an explanation of how to use this first.
* It seems we could simplify the explanation about the conventional reloading feature.

```suggestion
You can use this feature to reload config and restart Fluentd gracefully.

You can use this feature by sending a `SIGUSR2` signal to the supervisor process,
and this changes behavior `SIGUSR2` from graceful reload to zero-downtime-restart feature on Linux and macOS environments.

You can still use the conventional graceful reloading feature by directly sending `SIGUSR2` to each worker process or using the RPC feature.
However, it had some limitations and issues (Please see [#4624](https://github.com/fluent/fluentd/pull/4624) for details).
So, we recommend using this new feature for graceful reloading/restarting unless there is a particular reason.
```